docs/levenscyclus/index.md

```diff
@@ -1,32 +1,37 @@
 --- 
 title: Levenscyclus
+summary: Hulp per fase van de levenscyclus van algoritmes en AI, zoals probleemanalyse, ontwerpen, ontwikkelen, implementeren en ermee stoppen.
 icon: material/reload
 --- 
+# Levenscyclus algoritmes en AI
 
-!!! info "Disclaimer"
+Verantwoordelijke algoritmes en AI voldoen tijdens elke fase van de levenscyclus aan de regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
 
```

Comment:
```suggestion
Wanneer algoritmes gebruikt worden doorloop je als organisatie verschillende stappen. 
Het algoritme moet worden ontwikkeld, en na validatie wordt het algoritme in productie genomen. 
Na enige tijd kan besloten worden het gebruik te beëindigen. 
Bij verantwoord gebruik van algoritmes zul je tijdens iedere fase van de levenscyclus moeten zorgen dat je voldoet aan de vereisten en voldoende maatregelen hebt getroffen. 
```

Reply:
Bij nader inzien vond ik de tekst toch iets te kort. Ik heb gepoogd iets meer context toe te voegen

Reply:
Heb het net even anders opgeschreven. Kun je je er zo in vinden?
```suggestion
# Levenscyclus algoritmes en AI
Bij verantwoord gebruik van algoritmes en AI voldoe je tijdens elke fase van de levenscyclus aan de juiste regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.

## Hulp per fase
Overheden die werken met algoritmes of AI, doorlopen een aantal fasen. Van het ontwikkelen van een oplossing, tot het in gebruik nemen van die oplossing en er uiteindelijk weer mee stoppen. Voor dit hele proces gelden regels. 

De levenscyclus helpt je om te bepalen in welke fase jouw product zit en welke beslissingen hierbij horen. Per fase vind je een overzicht van de vereisten en aanbevolen maatregelen en instrumenten.

## Fases van de levenscyclus
```

Reply:
Wat wordt dan de tekst onder het kopje # Levenscyclus algoritmes en AI?

Wordt het dan niet dubbelop?

Reply:
Ha ha, ja mee eens. Het was een compromis-oplossing van me omdat ik dacht dat je liever wel extra uitleg wilde. 

Maar in dat geval... wat dacht je hiervan (geen extra alinea, maar een iets langer intro):

# Levenscyclus algoritmes en AI
Bij verantwoord gebruik van algoritmes en AI voldoe je tijdens elke fase van de levenscyclus aan de juiste regels. Van het ontwikkelen van een oplossing, tot het in gebruik nemen van die oplossing en er uiteindelijk weer mee stoppen. Voor dit hele proces gelden regels. De levenscyclus helpt je om te bepalen in welke fase jouw product zit en welke beslissingen hierbij horen.

## Fases van de levenscyclus
Etc.